Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -565,10 +565,10 @@ std::map<std::string, FieldValue> DecodeObject(pb_istream_t* stream) {
 
 }  // namespace
 
-void Serializer::EncodeFieldValue(const FieldValue& field_value,
-                                  std::vector<uint8_t>* out_bytes) {
-  Writer stream = Writer::Wrap(out_bytes);
-  EncodeFieldValueImpl(&stream, field_value);
+void Serializer::WriteFieldValue(const FieldValue& field_value,
```

Comment:
The verbs on the Serializer are Encode/Decode

cf Android: http://google3/java/com/google/android/gmscore/dev/thick_client/firebase-firestore/src/com/google/firebase/firestore/remote/RemoteSerializer.java?rcl=185935790

Basically within the context here I think we should end up with Writer writes and Serializer encodes. The free functions in here seem like they're private members of the serializer so I think they're "encode" too.

Reply:
Whoops. Yeah, that's why I used 'Encode' in the first place. (Apparently, I got a little carried away.) Fixed.

(And yeah, I agree that the free functions are also "encode".)